array-manipulation-2/src/drop-right.js

```diff
@@ -0,0 +1,23 @@
+/* exported dropRight */
+
+/*
+- create storage for output
+- look at each element one by one
+- if array length is greater than count
+  - add elements to output until count number away from the last element
+- otherwise,
+  - make output equal to array
+- return output
+*/
+
+function dropRight(array, count) {
+  var output = [];
+  for (var i = 0; i < array.length - count; i++) {
+    if (array.length > count) {
+      output.push(array[i]);
+    } else {
+      output = array;
```

Comment:
I would do the check if the array length is not as long as the `count` before you start the loop. If the size is less than the `count`, then you should just return an empty array.